test/modules/DirectMethodSender/ReporterClients/EventReporterClient.cs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace DirectMethodSender
+{
+    using System;
+    using System.Net;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil.TestResults;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    internal sealed class EventReporterClient : ReporterClientBase
```

Comment:
remove "internal", I think internal is the default access modifier if not specify

Reply:
Nope, it's private. I tested it earlier

Reply:
Check this out: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/access-modifiers#class-and-struct-accessibility; default access modifier of class is internal.